bga_src/backend/states.inc.php

```diff
@@ -138,9 +138,13 @@
 
   51 => [
     "name" => "endRound",
-    "type" => "game",
+    "descriptionmyturn" => clienttranslate(
+      'Waiting for all the players to confirm the result'
```

Comment:
```suggestion
      'Press Confirm to continue.'
```
Also add `description` with the message `Waiting for the opponent to confirm the result.